xbmc/addons/AddonDatabase.cpp

```diff
@@ -759,30 +759,6 @@ bool CAddonDatabase::Search(const std::string& search, VECADDONS& addons)
   return false;
 }
 
-void CAddonDatabase::SetPropertiesFromAddon(const AddonPtr& addon,
-                                           CFileItemPtr& pItem)
-{
-  pItem->SetProperty("Addon.ID", addon->ID());
-  pItem->SetProperty("Addon.Type", TranslateType(addon->Type(),true));
-  pItem->SetProperty("Addon.Name", addon->Name());
-  pItem->SetProperty("Addon.Version", addon->Version().asString());
-  pItem->SetProperty("Addon.Summary", addon->Summary());
-  pItem->SetProperty("Addon.Description", addon->Description());
-  pItem->SetProperty("Addon.Creator", addon->Author());
-  pItem->SetProperty("Addon.Disclaimer", addon->Disclaimer());
-  pItem->SetProperty("Addon.Rating", addon->Stars());
-  std::string starrating = StringUtils::Format("rating%d.png", addon->Stars());
-  pItem->SetProperty("Addon.StarRating",starrating);
-  pItem->SetProperty("Addon.Path", addon->Path());
```

Comment:
@tamland you're removing the addon.path & addon.id properties, but forgot to add those as listitem labels?

forum report: http://forum.kodi.tv/showthread.php?tid=270371